### PR TITLE
Dashboard: Fixes popovers everywhere

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
**Fixes**

- Filter dropdown always open
- Remove horizontal overflow from hidden menus
- Dynamic card popovers corner alignment wasn't working

**Technical updates**

- Ties the reveal portion of the component into all exported components
- Adds for base and variations pattern with popover reveals to allow for multiple reveal animations
- Separates out card popover from a standard popover so we can have different reveal and logic on the cards, but not bloat the components interface
- Centers all standard popovers
- Remove overflow from Dashboard area

****

### CODE REVIEW

File: assets/src/dashboard/components/popoverMenu/popoverCard.js [hunk 1]
```diff
@@ -68,27 +57,23 @@ const MenuCounterRevealer = styled.div`
...
+const ButtonInner = styled(Popover)`
   ${fullSize}
-  position: absolute;
-  z-index: ${Z_INDEX.POPOVER_MENU};
-  opacity: ${({ isOpen }) => (isOpen ? 1 : 0)};
-  pointer-events: ${({ isOpen }) => (isOpen ? 'auto' : 'none')};
   transform: ${(props) => {
     switch (props.align) {
-      case CORNER_DIRECTIONS.TOP_LEFT:
+      case CORNER_DIRECTIONS.top_left:
```
Comment (mariano-formidable) [inline assets/src/dashboard/components/popoverMenu/popoverCard.js:64]: Took me a little bit to realize why these had to be lowercase, but, ha, I get it! 👍

**Review by mariano-formidable — CHANGES_REQUESTED** [review]
The code looks good to me, but I noticed that the right side of "Last modified" sort order dropdown is being cut off by the screen.  I don't recall that happening before 🤔 Is that an "easy" fix?

image

**Review by BrittanyIRL — COMMENTED** [review]
Everything works! 

Second @mariano-formidable - and would add that when the sort dropdown now goes to the left side on smaller view ports it's butting up against the screen edge as well. Looks like the margin is no longer being respected on the page.  

I also think there's a test that needs updating re your structure changes - that's why the build's failing.

File: assets/src/dashboard/components/popoverMenu/popoverCard.js (hunk 1 above)
Comment (maxyinger) [reply assets/src/dashboard/components/popoverMenu/popoverCard.js:64]: lol same

**Review by carloskelly13 — APPROVED** [review]
Tested it out and I can now click on stuff and it is not cut off. This also fixes the layouts in smaller viewports.

File: assets/src/dashboard/app/views/shared/bodyViewOptions.js [hunk 2]
```diff
@@ -33,33 +33,19 @@ const DisplayFormatContainer = styled.div`
   display: flex;
   align-items: space-between;
   align-content: center;
-
-  @media ${({ theme }) => theme.breakpoint.smallDisplayPhone} {
-    flex-direction: column;
-    align-items: flex-start;
```
Comment (BrittanyIRL) [inline assets/src/dashboard/app/views/shared/bodyViewOptions.js:39]: I'm glad you did this. I like it more this way.

**Review by BrittanyIRL — APPROVED** [review]
Menus all work again and the style updates are a great add. Thank you.

**Review by mariano-formidable — APPROVED** [review]
Lookin' good! 🎉

**Pull request merged**